airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/sent_to.json

```diff
@@ -0,0 +1,79 @@
+{
+  "type": "object",
+  "title": "Sent To",
+  "description": "A list of member's subscriber activity in a specific campaign.",
+  "properties": {
+    "campaign_id": {
+      "type": "string",
+      "title": "The unique id for the campaign.",
+      "description": "The unique id for the campaign."
+    },
+    "list_id": {
+      "type": "string",
+      "title": "The unique id for the list.",
+      "description": "The unique id for the list."
+    },
+    "list_is_active": {
+      "type": "boolean",
+      "title": "The status of the list used.",
+      "description": "The status of the list used, namely if it's deleted or disabled."
+    },
+    "email_id": {
+      "type": "string",
+      "title": "email MD5 hash.",
+      "description": "The MD5 hash of the lowercase version of the list member's email address."
+    },
+    "email_address": {
+      "type": "string",
+      "title": "Email address for a subscriber.",
+      "description": "Email address for a subscriber."
+    },
+    "status": {
+      "type": ["string", "null"],
+      "title": "action",
+      "enum": ["sent", "hard", "soft"],
+      "description": "The status of the member ('sent', 'hard' for hard bounce, or 'soft' for soft bounce)."
+    },
```

Comment:
I see the `email_address` type doesn't contain `null`, but the `status` type does contain `null`; is this because `email_address` is a required value?